packages/frontend-2/components/viewer/anchored-point/NewThread.vue

```diff
@@ -69,6 +69,21 @@
             </div>
           </div>
         </div>
+        <div
+          v-if="modelValue.isExpanded && !canPostComment"
+          class="bg-foundation shadow-lg p-4 rounded-md flex flex-col items-center justify-center text-center"
+        >
+          <p class="text-sm mb-2 font-bold">Join the conversation</p>
+          <FormButton size="sm" @click="showLoginDialog = true">
+            Sign in to Speckle
+          </FormButton>
+          <AuthLoginPanel
+            v-model:open="showLoginDialog"
+            dialog-mode
+            max-width="sm"
+            subtitle="Create a free account to keep using Speckle!"
+          />
+        </div>
```

Comment:
I think this one is unnecessary. Let's just open the sign in dialog as soon as you click the new thread button above.

<img width="291" alt="Screenshot 2024-02-02 at 15 51 46" src="https://github.com/specklesystems/speckle-server/assets/2694102/40c191f1-c5dd-428e-8566-59063f3f2ddb">